qiskit/test/mock/fake_openpulse_2q.py

```diff
@@ -63,7 +65,10 @@ def __init__(self):
             conditional_latency=[
                 [100, 1000], [1000, 100], [100, 1000],
                 [1000, 100], [100, 1000], [1000, 100]
-            ]
+            ],
```

Comment:
I think I prefer if you revert changes to lines 68-102 in this file, because they should get updated in the backend config and pulse defaults PRs